CONTRIBUTING.md

```diff
@@ -27,9 +27,8 @@ the [puppet-dev mailing list](https://groups.google.com/forum/#!forum/puppet-dev
 
 ## Getting Started
 
-* Make sure you have a [Jira account](https://tickets.puppetlabs.com).
 * Make sure you have a [GitHub account](https://github.com/signup/free).
-* Submit a Jira ticket for your issue if one does not already exist.
+* (Optional) Submit a GitHub Issue if one does not already exist.
   * Clearly describe the issue including steps to reproduce when it is a bug.
   * Make sure you fill in the earliest version that you know has the issue.
   * A ticket is not necessary for [trivial changes](#making-trivial-changes) like a small typo fix.
```

Comment:
since we're making the GH issue optional, the trivial change section is no longer needed.